src/twisted/internet/base.py

```diff
@@ -1368,8 +1368,9 @@ def _handleSignals(self) -> None:
         signal.signal(signal.SIGTERM, reactorBaseSelf.sigTerm)
 
         # Catch Ctrl-Break in windows
-        if hasattr(signal, "SIGBREAK"):
-            signal.signal(signal.SIGBREAK, reactorBaseSelf.sigBreak)
+        SIGBREAK = getattr(signal, "SIGBREAK", None)
```

Comment:
Yeah, I noticed that this workaround was necessary to appease mypy, since
`signal.SIGBREAK` is only on Windows.

Reply:
Right, I'm trying not to add platform checks when checking for the attribute seems more robust.

Same sort of deal as we ran into with `select` module attributes.